helm/ucentral/values.ucentral-qa.haproxy-enabled.yaml

```diff
@@ -0,0 +1,49 @@
+owgw:
+  services:
+    owgw:
+      type: ClusterIP
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-type: "none"
```

Comment:
Shouldn't using null unset the value completely?

Reply:
Tried that - deployment fails with error:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: unknown object type "nil" in Service.metadata.annotations.service.beta.kubernetes.io/aws-load-balancer-type
```